package.json

```diff
@@ -107,7 +107,7 @@
     "@types/react-redux": "7.1.9",
     "@uniswap/sdk": "3.0.3",
     "@uniswap/v2-core": "1.0.1",
-    "@unstoppabledomains/resolution": "5.0.2",
+    "@unstoppabledomains/resolution": "7.1.4",
```

Comment:
100% sure no breaking changes? Asking because it's 2 major version bump

Reply:
yeah tested importing + send we gucci